src/Microsoft.AspNetCore.Razor.Design/Microsoft.AspNetCore.Razor.Design.csproj

```diff
@@ -2,8 +2,68 @@
 
   <PropertyGroup>
     <Description>Razor is a markup syntax for adding server-side logic to web pages. This package contains MSBuild support for Razor.</Description>
-    <TargetFrameworks>net46;netstandard2.0</TargetFrameworks>
+
+    <!-- 
+      Using netcoreapp2.0 here is a workaround for https://github.com/Microsoft/msbuild/issues/2661
+
+      We need to use a P2P reference to build some other projects that need to target netcoreapp2.0
+      and even though we don't actually want to reference the output, MSBuild will not allow it.
+    -->
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+
+    <!-- This project doesn't have any code, so don't include it in the .nupkg -->
+    <IncludeBuildOutput>false</IncludeBuildOutput>
   </PropertyGroup>
 
+  <!-- 
+    We're using CopyToOutputDirectory and Link here to mimic shape of what goes into the package in the output
+    directory. This is used by tests.
+  -->
+  <ItemGroup>
+    <Content Include="build\**\*.props" PackagePath="build\" />
+    <Content Include="build\**\*.targets" PackagePath="build\" />
+    <Content Include="buildMultiTargeting\*.props" PackagePath="buildMultiTargeting\" />
+    <Content Include="buildMultiTargeting\*.targets" PackagePath="buildMultiTargeting\" />
+
+    <Content Include="..\Microsoft.AspNetCore.Razor.Tasks\bin\$(Configuration)\net46\Microsoft.AspNetCore.Razor.Tasks.dll">
```

Comment:
Using the net46 reference is currently broken right now because the P2P reference doesn't build twice. I'm going to remove net46 for now since it's not on the critical path and log an issue. I'll put my head together with @natemcmaster and @pranavkm and see if we can figure this out next week.

Reply:
Logged #1843 to track this.